wrappers/golang/internal/generator/curves/templates/curve.go.tmpl

```diff
@@ -43,9 +43,15 @@ func (p *{{.CurvePrefix}}Projective) FromAffine(a {{.CurvePrefix}}Affine) {{.Cur
 	z := {{.CurvePrefix}}BaseField{}
 	z.One()
 
+	if (a.X == z.Zero()) && (a.Y == z.Zero()) {
+		p.X = z.Zero()
+		p.Y = z.One()
+		p.Z = z.Zero()
```

Comment:
Better to use the Zero function from the Projective point directly so we don't have multiple locations of its internal logic
```suggestion
		p.Zero()
```